src/miner.cpp

```diff
@@ -76,6 +80,9 @@ static BlockAssembler::Options DefaultOptions()
     } else {
         options.blockMinFeeRate = CFeeRate(DEFAULT_BLOCK_MIN_TX_FEE);
     }
+
+    options.m_skip_inclusion_until = std::chrono::microseconds::max();
+    options.check_block_validity = true;
```

Comment:
These aren't doing anything. They already get set in the `Options` constructor.

Reply:
good point, removed